### PR TITLE
Add subscription migration logic

### PR DESCRIPTION
**Pull request by rudigiesler**
This PR is concerned with adding the logic to migrate an identity between the specified messagesets.

### CODE REVIEW

File: mapper/sequence_mapper.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+
+class SequenceMapper(object):
+    """
+    Provides the logic for mapping the sequence number from one message set to
+    another.
+    """
+    def map(self, from_messageset, to_messageset, sequence):
```
Comment (erikh360) [inline mapper/sequence_mapper.py:10]: This doesn't do much, is it a placeholder for logic that will be added later?

**Review by erikh360 — COMMENTED** [review]
Looks good, just the one question.

File: mapper/sequence_mapper.py (hunk 1 above)
Comment (rudigiesler) [reply mapper/sequence_mapper.py:10]: Yup. When the gates message sets have been finalised, then we can add the mapping logic here on how to go back and forth between the two sets.

Comment (erikh360) [reply mapper/sequence_mapper.py:10]: Cool, then I'm +1

**Review by erikh360 — APPROVED** [review]

**Pull request merged**